src/d3a/models/area/__init__.py

```diff
@@ -209,7 +209,7 @@ def get_grid_fee(self):
         grid_fee_type = self.config.grid_fee_type \
             if self.config is not None \
             else ConstSettings.IAASettings.GRID_FEE_TYPE
-        return self.get_path_to_root_fees() if grid_fee_type == 1 else 0
+        return self.grid_fee_constant if grid_fee_type == 1 else self.grid_fee_percentage
```

Comment:
Maybe @lana-shanghai can help more here, but I think this is deliberate. IIRC Sarah mandated to display 0 if the grid fee type is percentage.

Reply:
@spyrostz get_grid_fees is only used for the external endpoints (so for the api clients) They dont want to have the gridfee of the whole tree included.

Reply:
@hannesdiedrich you are right, I misread the function name's intent